scripts/getDependencies.pl

```diff
@@ -116,6 +116,14 @@
 	sha1 => 'c9ad4a0850ab676c5c64461a05ca524cdfff59f1'
 );
 
+my %tohandler_simple = (
+	url => 'https://ci.eclipse.org/openj9/view/Infrastructure/job/Build_JDK_Timeout_Handler/lastSuccessfulBuild/artifact/openj9jtregtimeouthandler.jar',
+	fname => 'openj9jtregtimeouthandler.jar',
+	shaurl => 'https://ci.eclipse.org/openj9/view/Infrastructure/job/Build_JDK_Timeout_Handler/17/artifact/openj9jtregtimeouthandler.jar.sha256sum.txt',
```

Comment:
Should this be `lastSuccessfulBuild `?

https://ci.eclipse.org/openj9/view/Infrastructure/job/Build_JDK_Timeout_Handler/lastSuccessfulBuild/artifact/openj9jtregtimeouthandler.jar.sha256sum.txt

Reply:
Doh, updated.